pkgs/development/pharo/launcher/default.nix

```diff
@@ -52,7 +51,7 @@ stdenv.mkDerivation rec {
      secs=5
      echo -n "Running headless Pharo for $secs seconds to check for a crash... "
      timeout $secs \
-       ${pharo-vm}/bin/pharo-vm-nox PharoLauncher.image --no-quit eval 'true'
+       "${pharo}/bin/pharo" -nodisplay PharoLauncher.image --no-quit eval 'true'
```

Comment:
Possible minor typo: At least on my machine, `pharo` complains about the `-nodisplay` argument, using two dashes (`--nodisplay`) does the trick.